src/Components/ArticleById.jsx

```diff
@@ -6,11 +6,14 @@ import {
 import { useParams } from "react-router";
 import { Link } from "react-router-dom";
 import { v4 as uuidv4 } from "uuid";
+import { upateCommentVotes } from "../Utilis/PatchVote";
+
 export default function ArticleById() {
   const [commentvisble, setCommentVisble] = useState(false);
   const [articles, setArticles] = useState([]);
   const [articlesCom, setArticlesCom] = useState([]);
   const [loading, setLoading] = useState(false);
+  const [votesNo, setVotesNo] = useState(0);
```

Comment:
your votesNo state should be set to the current articles number of votes rather than 0